src/components/LoadingShape/index.js

```diff
@@ -0,0 +1,58 @@
+import React, { useEffect, useRef } from 'react';
+import PropTypes from 'prop-types';
+import RenderIf from '../RenderIf';
+import {
+    StyledShapeContainer,
+    StyledLoadingShape,
+    StyledImageIcon,
+    StyledAvatarIcon,
+} from './styled';
+
```

Comment:
Let's add a description for the component

Reply:
Done